### ISSUE TITLE
[Feature]: Upload iOS LivePhoto downloaded from icloud

### ISSUE THREAD

**Issue by nullket** [immich-app/immich#1196]
### Feature detail

As part of my old photo management I simply downloaded all my photos from the iCloud. For LivePhotos this results in a `.JEPG` and a `.MOV` file. Uploading those two files via the web interface does NOT result in a single LivePhoto in immich but remains separated as two entries: a small video and a still image. 

### Platform

Web

**Comment by alextran1502** [comment]
Yes, currently, the LivePhotos upload is limited to the mobile app. 

I will put this feature into the todo list

**Comment by PeterBasista** [comment]
@nullket if you don't want to wait, you can still link them yourself. I mentioned it in this [comment](https://github.com/immich-app/immich/issues/1133#issuecomment-1361953894) .

**Comment by nullket** [comment]
> @nullket if you don't want to wait, you can still link them yourself. I mentioned it in this [comment](https://github.com/immich-app/immich/issues/1133#issuecomment-1361953894) .

Thanks! I was already looking for a workaround. I will have a look into this (looks a bit challenging as I am not using direct sql queries very often).

**Comment by erikibolya** [comment]
On Discord, i proposed a method of coupling Live Photo files together by its ContentIdentifier attribute.

Apple generates unique ids for image and video part of Live Photo
(for the reference: https://stackoverflow.com/a/47629868/3384936). 

With this in mind, there should be a way of uploading parts of Live Photos separately and Immich should be able to pair them on Database level. 

For example: store ContentIdentifier as a column in DB and use JOINS to generate pairs of photo and video

Copy of Live Photo File has the same ContentIdentifier so this could also identify duplicates of these photos (nice little side effect?)

**Comment by nep2ner** [comment]
Did anyone end up finding a reliable to way to get photos from iCloud Photos into Immich, while preserving the live photos?

**Comment by nep2ner** [comment]
Maybe there is a way to download the photos from iCloud to the iPhone in batches and let the Immich app do the uploading?

**Comment by nullket** [comment]
If you still have them in the iCloud just sync them to the iOS app and upload them with the iOS Immich app. Immich supports Live Photos well if you upload them using the app. 

I had previously downloaded Live Photos from the iCloud via the browser and then deleted the originals from the cloud. This resulted in two local files (a jpg and a mov). Unfortunately, reuploading them to the icloud did not work. But the original macOS photo app imports those two files and recreates a Live Photo as expected, using the iCloud photo sync I was then able to resync a real iOS Live Photos back on my iPhone where I finally used Immich to upload everything. A bit of back and forth but it worked for me without being scarred of breaking my database with custom sql queries, I do not fully understand.
